src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx

```diff
@@ -31,5 +31,30 @@ You will also need to install the pod dependencies for iOS:
 ```sh
 npx pod-install
 ```
+
+**Using SQLite (optional)**
+
+React Native CLI apps can now use SQLite with DataStore. SQLite offers considerable performance advantages over AsyncStorage, the default database.
+
+At this time, SQLite is an opt-in feature, and you can enable it with the following steps:
+
+```
```

Comment:
```suggestion
```sh
```
is preventing the build from succeeding 😄

Reply:
@tjleing it appears there are other markdown issues from code that's already in the repo. E.g., this PR didn't change the file that's erroring out in the build:

![image](https://user-images.githubusercontent.com/29709626/131901568-6fe67c14-169e-4e43-ad24-c4df3130d912.png)